applications/nrf_desktop/src/hw_interface/Kconfig

```diff
@@ -117,6 +117,14 @@ config DESKTOP_WHEEL_SENSOR_VALUE_DIVIDER
 	  A value obtained from the wheel sensor will be divided
 	  by provided argument before being processed further.
 
+config DESKTOP_WHEEL_SENSOR_IDLE_TIMEOUT
+	int "Wheel sensor idle timeout"
+	depends on DESKTOP_WHEEL_ENABLE
+	default 0
```

Comment:
add range ?

Reply:
I don't think it would be needed. 0 disables the feature (documentation updated) and all other values are valid (though may be stupid). It may be that some values will not be valid but this is more complicated as it depends on the length of a tick and type used to stored time. For sane values it should be fine.

Reply:
In the end I set the range and limited the value to 1 hour (3600 s). It should be high enough to be impractical to set and secures against the overflow problems on time calcs